### ISSUE TITLE
About retrain topiq model

### ISSUE THREAD

**Issue by cyclopssssss** [chaofengc/IQA-PyTorch#109]
use your training options of TOPIQ on koniq dataset like below: 
python pyiqa/train.py -opt options/train/TOPIQ/train_TOPIQ_res50_koniq.yml
and the result is bit lower than the benchmark you provided:
![M~SQJL9A@D%WN1FOV81Z1DO](https://github.com/chaofengc/IQA-PyTorch/assets/50789512/cd897db5-cd95-4142-a7fc-2cdd80ce0ec7)
Could you give some advice about training the model?🥺
Thanks for your patience!

**Comment by chaofengc** [comment]
Thanks for your information. I have checked the training history and updated the loss function as below
https://github.com/chaofengc/IQA-PyTorch/blob/81edc661f1ea13ccd7d0cec8a9518f377155f9da/options/train/TOPIQ/train_TOPIQ_res50_koniq.yml#L82-L88

You may refer to the paper [Norm-in-Norm Loss with Faster Convergence and Better Performance for Image Quality Assessment
](https://arxiv.org/abs/2008.03889) for more details.

**Comment by cyclopssssss** [comment]
Thanks again for your reply!
